src/ExpressionEvaluator/CSharp/Test/ResultProvider/ValueFormattingTests.cs

```diff
@@ -214,8 +214,8 @@ public void String()
             Assert.Equal("\"\\uffff\"", FormatValue(s, useHexadecimal: true));
 
             string multiByte = "\ud83c\udfc8";
-            Assert.Equal(string.Format(format, "🏈"), FormatValue(multiByte));
-            Assert.Equal(string.Format(format, "🏈"), FormatValue(multiByte, useHexadecimal: true));
+            Assert.Equal(string.Format(format, "\\ud83c\\udfc8"), FormatValue(multiByte));
+            Assert.Equal(string.Format(format, "\\ud83c\\udfc8"), FormatValue(multiByte, useHexadecimal: true));
```

Comment:
> The justification is that the implementation is incorrect, and the test carefully tests only one case for which the incorrect implementation happens to work.

The implementation is incorrect but people may be depending on it. Should we be taking this through compat council?